### ISSUE TITLE
doesn't seem to be working

### ISSUE THREAD

**Issue by deutrino** [meitar/cross-pod-links#2]
This extension is not working for me with Tampermonkey installed.

**Comment by fabacab** [comment]
What error are you seeing?

**Comment by deutrino** [comment]
There's no hover or click-to-activate bubble as advertised - there is one from D\* itself, though.

I suppose gathering some console output might be good. I'll try to do that.

**Comment by fabacab** [comment]
It would also be helpful to know what Diaspora pod you are using, of course….

**Comment by deutrino** [comment]
I maintain testing accounts on several pods, currently I'm seeing it on diasp.de and nerdpol.ch, will check others when I have a minute :)

**Comment by fabacab** [comment]
It's still working on JoinDiaspora.com for me. It's probably no longer working for pods using the new Diaspora version 0.5.1. Can you please check to see if that's a consistent characteristic on all the pods where this script currently fails? Many thanks.

**Comment by deutrino** [comment]
Failing (or PEBKAC) on:
- 0.5.1.1-pfe7882ed (diasp.de)
- 0.4.1.3 (librenet.gr, not listed in sidebar, but here's the [github](https://github.com/librenet/librenet.gr))
- 0.5.1.1-p80ea6b41 (nerdpol.ch)
- 0.4.1.3-pca46b457 (pod.readme.is)

**Comment by deutrino** [comment]
Every one of these pods already has a popup when you mouseover the "n hours ago" thingy, which displays the date. Maybe that's not on joindiaspora.com yet and is thus interfering with your popup?

**Comment by fabacab** [comment]
Nah, the "pop-up" is a tooltip, and this doesn't interfere with that.

I'll look at the other pods when I get a chance ti see if there's something else going on. I know that version 0.5.x changed the markup in some ways so that's bound to break. But the 0.4.x ones "should" work.

**Comment by deutrino** [comment]
Thanks. I'm kind of at the "desperate flailing" point in my scheduling but I'll try to poke at it with Chrome dev tools etc when I have a little time.

**Comment by fabacab** [comment]
Yeah, same. I can't _promise_ I'll spend any time on this until/unless it begins breaking on my pod. :)

**Comment by fabacab** [comment]
Hmm. My pod just updated to 0.5.1.2 and I have not noticed anything breaking. I still get the correct pop-overs and the correct cross-pod links. :\ I am on JoinDiaspora.com, FWIW. Each of the pods that you listed were running different Diaspora versions. Maybe see if this issue resolves itself for you when they update to the same codebase that my pod just updated to?

**Comment by SunyataZero** [comment]
Hi and thank you for creating this script, unfortunately it doesn't work for me either, nothing different happens when i click on the date link for the post, i just get to the post as usual

I am on diasp.eu and i've installed the "Tampermonkey" plugin for Chrome before adding your JS

Kind Regards, Tord
